db/migrate/20240624121212_katello_recreate_evr_constructs.rb

```diff
@@ -0,0 +1,159 @@
+class KatelloRecreateEvrConstructs < ActiveRecord::Migration[6.1]
+  def change
+    count = select_value <<~SQL
+      SELECT count(*) FROM pg_extension WHERE extname = 'evr';
+    SQL
+    if count.to_i == 0
```

Comment:
In the native DSL there's both `extension_available?` and `extension_enabled?`. I think the latter was intended here:
```suggestion
    if !extension_enabled?('evr')
```